ledger/accountdb.go

```diff
@@ -1192,6 +1195,18 @@ func accountsInit(tx *sql.Tx, initAccounts map[basics.Address]basics.AccountData
 		if err != nil {
 			return true, err
 		}
+
```

Comment:
this is schema v0 init, and should not set any new data. Well, it can theoretically but check all all other new tables creation like resources and onlineaccounts